src/core/providers/qgsabstractdatabaseproviderconnection.cpp

```diff
@@ -1340,6 +1340,21 @@ QList< QgsWeakRelation > QgsAbstractDatabaseProviderConnection::relationships( c
   return {};
 }
 
+void QgsAbstractDatabaseProviderConnection::addRelationship( const QgsWeakRelation & ) const
+{
+  checkCapability( Capability::AddRelationship );
```

Comment:
A generic question about unused arguments, I always though it was a QGIS coding convention to explicitly indicate that an argument is not used by marking it with `Q_UNUSED`, do we have a preference?

I'm happy whatever we decide, perhaps Q_UNUSED is more visible and explicit than omitting the argument name.

Reply:
There's no convention really. I tend slightly toward an "avoid qt specific stuff when it's painless to do so", and so don't really see the need for Q_UNUSED vs plain old C++ ways to hide this warning...